fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -949,4 +958,18 @@ public Expression visitExist(ExistContext context) {
     public List<Expression> withInList(PredicateContext ctx) {
         return ctx.expression().stream().map(this::getExpression).collect(ImmutableList.toImmutableList());
     }
+
+    static class SelectQuerySpecificationContext {
```

Comment:
if u need it, the better way to do this is add a new Rule include `whereClause? aggClause? havingClause?` named `SelectQuerySpecification` in `DorisParser.g4`